pkg/fakerp/manifest.go

```diff
@@ -0,0 +1,46 @@
+package fakerp
+
+import (
+	"bytes"
+	"html/template"
+	"os"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+
+	sdk "github.com/openshift/openshift-azure/pkg/util/azureclient/osa-go-sdk/services/containerservice/mgmt/2018-09-30-preview/containerservice"
+)
+
+func readEnv() map[string]string {
+	env := make(map[string]string)
+	for _, setting := range os.Environ() {
+		pair := strings.SplitN(setting, "=", 2)
+		env[pair[0]] = pair[1]
+	}
+	return env
+}
+
+// GenerateManifest returns the input manifest using the envirionment
+func GenerateManifest() (*sdk.OpenShiftManagedCluster, error) {
+	manifestFile := os.Getenv("MANIFEST")
+	if len(manifestFile) == 0 {
+		manifestFile = "test/manifests/normal/create.yaml"
```

Comment:
pass the default manifest in GenerateManifest as an arg

Reply:
Actually, do the defaulting in fakerp.Config and pass a single string here that is going to be read, parsed, and generated

Reply:
done